src/components/flat-table/flat-table.stories.mdx

```diff
@@ -108,9 +111,73 @@ import { SidebarContext } from '../drawer';
   </Story>
 </Preview>
 
+### With custom cell paddings
+
+Padding can be set on each cell individually by using the usual spacing props syntax.
```

Comment:
> the usual spacing props syntax 

This should be a link to some documentation. I guess we can't link to styled-system because we need to specifically document our theme multiplier.